### DISCUSSION TITLE
Commerce

### DISCUSSION THREAD

**Discussion by Bcdo** [trendyminds/isolate#36] [Ideas]
Is there a way to make this work with products in commerce? 

I am trying to gather a couple of people that wants to sell some designs, but they should not be able to see the products the other designers create. Products themselves I guess is just like entries, only with some more attributes to them as standard.

**Comment by aaronbushnell** [comment]
It's currently not on the radar but if you or any others would like to take a stab at it and shoot over a PR we'd certainly review it!

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
